packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -771,7 +771,7 @@ class SliverAppBar extends StatefulWidget {
   ///
   /// ```dart
   /// return new Scaffold(
```

Comment:
Just looked at the travis output, you'll also have to remove the return statement since that's not legal here. After that, Travis should be happy.